src/main/java/edu/ucla/library/services/metadata/MetadataSetter.java

```diff
@@ -328,6 +338,16 @@ private void addMetadata(final String... aRow) throws FileNotFoundException, FfP
             final FFmpegFormat format = probeResult.getFormat();
 
             aRow[aRow.length - DURATION_OFFSET] = String.valueOf(format.duration);
+            if (myCsvHeaders.hasFormatExtentIndex() && aRow[myCsvHeaders.getFormatExtentIndex()].trim().equals("")) {
+                final double rawDuration = format.duration;
+                final int hours = (int) rawDuration / ONE_HOUR;
+                final int minutes = (int) (rawDuration % ONE_HOUR) / ONE_MINUTE;
+                final int seconds = (int) rawDuration % ONE_MINUTE;
+                final StringBuffer formattedDuration =
+                    new StringBuffer( hours > 0 ? String.format("%02dh", hours) : "" );
+                formattedDuration.append(String.format(" %02dm %02ds", minutes, seconds));
+                aRow[myCsvHeaders.getFormatExtentIndex()] = formattedDuration.toString().trim();
```

Comment:
(I am interested in others' thoughts on this, whether that be for or against)

Reply:
I'm personally fine with the static call from the ternary. I might suggest creating the sb first and then appending the ternary:
```
final StringBuffer formattedDuration = new StringBuffer();

formattedDuration.append(hours > 0 ? String.format("%02dh", hours) : "");
```
to avoid the line wrap and separate the ternary from the constructor for simplicity (or I guess one could create a value from the ternary and add it to the StringBuffer via the constructor to achieve the same goal).

I don't have strong feelings about this though.

Reply:
It's not so much that I think we shouldn't be doing function calls within ternaries (within a constructor), but that having that level of nesting in this instance is making the line longer than 120 chars, which I think could be avoided in this case by also making this snippet easier to read. I feel like I have to spend a little more time reading this to understand it than if it were, say, an if-else instead where it would be more immediately obvious. Just my 2c.

Reply:
Modified/simplified code